dataland-frontend/src/components/general/UserProfileDropDown.vue

```diff
@@ -87,6 +87,12 @@ export default defineComponent({
           id: "profile-picture-dropdown-data-request-button",
           clickAction: this.gotoDataRequest,
         },
+        {
+          label: "QUALITY ASSURANCE",
+          icon: "add_moderator",
+          id: "profile-picture-dropdown-qa-services-button",
+          clickAction: this.gotoQualityAssurance,
+        },
         {
           label: "LOG OUT",
           icon: "logout",
```

Comment:
The `id` of the logout dropdown option is a duplicate of an already existing `id` in the HTML template (of the profile picture div).

Reply:
Done.